Source/WebKit/WebProcess/WebPage/Cocoa/UnifiedTextReplacementController.mm

```diff
@@ -891,10 +898,10 @@
     });
 }
 
-void UnifiedTextReplacementController::replaceContentsOfRangeInSession(const WTF::UUID& uuid, const WebCore::SimpleRange& range, WebCore::DocumentFragment& fragment)
+void UnifiedTextReplacementController::replaceContentsOfRangeInSession(const WTF::UUID& uuid, const WebCore::SimpleRange& range, WebCore::DocumentFragment& fragment, WebCore::Editor::MatchStyle matchStyle)
 {
-    replaceContentsOfRangeInSessionInternal(uuid, range, [&fragment](WebCore::Editor& editor) {
-        editor.replaceSelectionWithFragment(fragment, WebCore::Editor::SelectReplacement::Yes, WebCore::Editor::SmartReplace::No, WebCore::Editor::MatchStyle::No, WebCore::EditAction::InsertReplacement);
+    replaceContentsOfRangeInSessionInternal(uuid, range, [&fragment, matchStyle](WebCore::Editor& editor) {
```

Comment:
Won't you have to update the signature for `replaceContentsOfRangeInSessionInternal`? What do you do with the `matchStyle` arg?

Reply:
no; I use it within the lambda parameter (the next line of the diff)

Reply:
ah, my bad. I confused the lambda capture list with arguments for `replaceContentsOfRangeInSessionInternal`